plays/otempo.py

```diff
@@ -0,0 +1,71 @@
+import time
+
+from playwright.sync_api import sync_playwright
+
+from plays.base import BasePlay
+from plays.items import EntryItem
+from plog import logger
+
+
+class OTempoPlay(BasePlay):
+    name = "otempo"
+
+    @classmethod
+    def match(cls, url):
+        return "otempo.com.br" in url
+
+    def pre_run(self):
+        pass
+
+    def run(self) -> EntryItem:
+        with sync_playwright() as p:
+            browser = self.launch_browser(p, viewport={"width": 1920, "height": 1080})
+            page = browser.new_page()
+            logger.info(f"[{self.name}] Abrindo URL '{self.url}'...")
+            page.goto(self.url, timeout=180_000)
+            
+            # Aguarda o conteúdo principal ficar visível
+            page.wait_for_selector("h1", timeout=30000)
+            
+            # Extrai o título do artigo
+            entry_title = ""
```

Comment:
![Screenshot 2025-06-18 at 19 11 01](https://github.com/user-attachments/assets/9498f71d-6369-48e8-b158-c6a324b693f7)
play não está conseguindo pegar dados da notícia